mm1.py

```diff
@@ -1,38 +1,39 @@
 """
-'This code is so ugly it hurts my eyes.'
-Mathias Hillesheim, this code's father
+'Only God can judge me.'
 """
 
 import random
-from datetime import datetime
-from queue import Queue
-
 import numpy as np
 import matplotlib.pyplot as plt
 from math import log
-
+from beautifultable import BeautifulTable
+from datetime import datetime
+from queue import Queue
 
 class Task(object):
 
     def __init__(self, simulation_time):
         self.simulation_time = simulation_time
 
     def get_time(self):
-        return self.simulation_time
+        return self.simulation_tim
```

Comment:
I don't believe that this will work, `Task` doesn't know `simulation_tim`. Even if it's not used, either we fix it or remove it.